runHarness/Parameters.pm

```diff
@@ -2420,119 +2429,109 @@ $parameters{"msgServerSuffix"} = {
 	"showUsage" => 0,
 };
 
-$parameters{"weathervaneHome"} = {
-	"type"    => "=s",
-	"default" => "/root/weathervane",
-	"parent"  => "",
-	"usageText" =>
-"This is the base directory under which the Weathervane harness expects to find its files.\n\tIf other directory related parameters are not diven with a full\n\tpath, they will be realative to this directory.",
-	"showUsage" => 1,
-};
-
-# If not absolute (starting with /) these directories are relative
-# to weathervaneHome
+# These directories are relative to the current working directory
 $parameters{"tmpDir"} = {
 	"type"    => "=s",
 	"default" => "tmpLog",
 	"parent"  => "runProc",
 	"usageText" =>
-"This is the directory in which Weathervane stores temporary files during a run.\n\tDo not use /tmp or any directory whose contents you wish to keep.\n\tIt must either include the full path to the directory\n\tor be relative to weathervaneHome.",
+"This is the directory in which Weathervane stores temporary files during a run relative to the current working directory.\n\tDo not use any directory whose contents you wish to keep.",
 	"showUsage" => 0,
 };
 $parameters{"outputDir"} = {
 	"type"    => "=s",
 	"default" => "output",
 	"parent"  => "runProc",
 	"usageText" =>
-"This is the directory in which Weathervane stores the output from runs.\n\tIt must either include the full path to the directory\n\tor be relative to weathervaneHome.",
+"This is the directory in which Weathervane stores the output from runs relative to the current working directory.",
 	"showUsage" => 0,
 };
 $parameters{"sequenceNumberFile"} = {
 	"type"    => "=s",
-	"default" => "output/sequence.num",
+	"default" => "sequence.num",
 	"parent"  => "runProc",
 	"usageText" =>
-"This is the file that Weathervane uses to store the sequence number of the next run.\n\tIt must either include the full path to the file\n\tor be relative to weathervaneHome.",
+"This is the file that Weathervane uses to store the sequence number of the next run.",
 	"showUsage" => 0,
 };
 $parameters{"distDir"} = {
 	"type"    => "=s",
 	"default" => "dist",
 	"parent"  => "runProc",
 	"usageText" =>
-"This is the directory for the executables and other artifacts needed to run Weathervane.\n\tIt must either include the full path to the directory\n\tor be relative to weathervaneHome.",
+"This is the directory for the executables and other artifacts needed to run Weathervane relative to the current working directory.",
 	"showUsage" => 0,
 };
 $parameters{"dbScriptDir"} = {
```

Comment:
The dbScriptDir parameter and all other references to dbScriptDir can be eliminated because the dbScripts are not located in the postgresql docker image.